### PR TITLE
Added footnote to user secrets section

### PR DESCRIPTION
**Pull request by Oliverst8**

### CODE REVIEW

File: docs/report.md [hunk 1]
```diff
@@ -477,9 +477,11 @@ dotnet user-secrets set "authentication:github:clientId" "Ov23liOEFAiXHOnNGkH3"
 dotnet user-secrets set "authentication:github:clientSecret" "9cc3aae28d9e5fdfe27f42158842f92687964382"
 ```
 
-Now run ```dotnet run```. The program is now running locally, go to http://localhost:5273 to interact with it
+Now run ```dotnet run```. The program is now running locally, go to http://localhost:5273 to interact with it[^2]
 
 
+[^2]: Pushing secrets like these to a public github repo are a bad idea. They are added here to make it easier to run the program, without having to create your own github oauth token, and the program _Chirp!_ is not of the importance where this is a problem.
```
Comment (JonathanVil) [inline docs/report.md:483]: are -> is
pushing _is_ a bad idea

Comment (JonathanVil) [inline docs/report.md:483]: Please use proper noun casing:
github -> GitHub
oauth -> OAuth

File: docs/report.md [hunk 2]
```diff
@@ -477,9 +477,11 @@ dotnet user-secrets set "authentication:github:clientId" "Ov23liOEFAiXHOnNGkH3"
 dotnet user-secrets set "authentication:github:clientSecret" "9cc3aae28d9e5fdfe27f42158842f92687964382"
```
Comment (JonathanVil) [inline docs/report.md:477]: This overflows the page line in the PDF, consider a manual line break

**Review by iiinaD — CHANGES_REQUESTED** [review]
see comments

**Review by iiinaD — APPROVED** [review]
I approve do not edit this please x2

**Pull request merged**